### PR TITLE
Refactor to make the AppNexusConncetor the exported class

### PR DESCRIPTION
**Pull request by alextremp**
This PR includes:

* The exported class is AppNexusConnector
* The previous AppNexusConnectorImpl is renamed to AstClientImpl
* The previous AppNexusClient is renamed to AstWrapper

### CODE REVIEW

File: src/openads-appnexus/AppNexusConnector.js [hunk 1]
```diff
@@ -1,87 +1,91 @@
...
-    throw new Error('AppNexusConnector#clearRequest must be implemented')
-  }
-
-  /**
-   * Refreshes ads on the page.
-   * @param target : an array of ids
-   */
-  refresh (target) {
-    throw new Error('AppNexusConnector#refresh must be implemented')
+  refresh ({id, segmentation}) {
```
Comment (rafamarquezv) [inline src/openads-appnexus/AppNexusConnector.js]: Could you please add some JSDoc to all methods on this class in order to clarify the meaning of the parameters and the purpose of the method itself? Thanx!

**Review by rafamarquezv — CHANGES_REQUESTED** [review]
Nice refactor!!! just some JSDoc missing... 📚

File: src/openads-appnexus/AppNexusConnector.js (hunk 1 above)
Comment (rafamarquezv) [reply src/openads-appnexus/AppNexusConnector.js]: I see the requested info is included here: https://github.com/scm-spain/OpenAds-ConnectorAPI/pull/1/files

**Review by rafamarquezv — APPROVED** [review]
👏 👏 👏

**Review by pablovegau — APPROVED** [review]

**Review by victuxbb — APPROVED** [review]

**Pull request merged**